templates/style.css

```diff
@@ -0,0 +1,226 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap');
+body {
+    padding: 0;
+    margin: 0;
+    height: 95vh;
+    font-family: Lato;
+    box-sizing: border-box;
+}
+
+.container,
+.content,
+main {
```

Comment:
Why not put everything on a single file, the way we handle verification and email confirmation.
Both CSS and HTML are on the same page, secondly, you can convert image to base64 and include it.

Reply:
when I convert to base64, will I write it again to a file?

Reply:
```
<div>
  <p>Taken from wikpedia</p>
  <img src="data:image/png;base64, iVBORw0KGgoAAAANSUhEUgAAAAUA
    AAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO
        9TXL0Y4OHwAAAABJRU5ErkJggg==" alt="Red dot" />
</div>

```

Replace from the `iVBO...` to `ggg==`